### ISSUE TITLE
Deploy application on Heroku

### ISSUE THREAD

**Issue by mohitrajane** [mohitrajane/neeto-ui-challange-23#8]

**Comment by mohitrajane** [comment]
Deployed at: https://github.com/mohitrajane/neeto-ui-challange-by-mohit-rajan-e/issues

Login credentials:
Username: [oliver@example.com](mailto:oliver@example.com)
Password: welcome

@yedhink @navaneethsdk Please review

**Comment by navaneethsdk** [comment]
@mohitrajane Please provide the right link for the deployed website.

**Comment by navaneethsdk** [comment]
@mohitrajane A few comments:
- [ ] Make it a habit to put translations at the top of a component. https://github.com/mohitrajane/neeto-ui-challange-by-mohit-rajan-e/blob/9b2e15767820d6b59029149db64f38e48ba317ec/app/javascript/src/components/Dashboard/Notes/index.jsx#L24
- [ ] Is there any specific reason for using `searchTerm` as an intermediate variable? Not only does it consume memory but also it can cause confusion since the state name is also `searchTerm`. Also please expand `e` as `event`. https://github.com/mohitrajane/neeto-ui-challange-by-mohit-rajan-e/blob/9b2e15767820d6b59029149db64f38e48ba317ec/app/javascript/src/components/Dashboard/Notes/index.jsx#L27-L28
- [ ] It does seem right to use `get` prefix with this utils function since we are not technically getting anything. A name like `formatDateandTime` would be better. https://github.com/mohitrajane/neeto-ui-challange-by-mohit-rajan-e/blob/9b2e15767820d6b59029149db64f38e48ba317ec/app/javascript/src/components/Dashboard/Notes/Card.jsx#L44
- [ ] This is not the right way to use `filter`. You should have used `find` method instead. https://github.com/mohitrajane/neeto-ui-challange-by-mohit-rajan-e/blob/9b2e15767820d6b59029149db64f38e48ba317ec/app/javascript/src/components/Dashboard/Notes/Card.jsx#L56-L58
- [ ] Why have you not passed the `reset` type on the `Cancel` button? https://github.com/mohitrajane/neeto-ui-challange-by-mohit-rajan-e/blob/9b2e15767820d6b59029149db64f38e48ba317ec/app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx#L81
- [ ] Regarding the logistics, I believe it was communicated in the Basecamp post that all PR should be assigned to reviewers after merging them. Let me know if you missed that point or if it wasn't clear enough. I'll update the doc accordingly if the document needs more clarity.

**Comment by mohitrajane** [comment]
@navaneethsdk 

> Make it a habit to put translations at the top of a component. https://github.com/mohitrajane/neeto-ui-challange-by-mohit-rajan-e/blob/9b2e15767820d6b59029149db64f38e48ba317ec/app/javascript/src/components/Dashboard/Notes/index.jsx#L24


In the [React best practices book](https://academy.bigbinary.com/learn-react/react-best-practices/structuring-code#keep-usestate-at-top), it is mentioned that `useState()` should come first. However, should this rule be broken for `useTranslation()` since it is used in most of the React components?

> Is there any specific reason for using searchTerm as an intermediate variable? Not only does it consume memory but also it can cause confusion since the state name is also searchTerm. Also please expand e as event. 

No specific reason will update.


> Why have you not passed the reset type on the Cancel button? 

The functionality was working without it, and since we didn't need the reset functionality, I didn't include it as a parameter.


> Regarding the logistics, I believe it was communicated in the Basecamp post that all PR should be assigned to reviewers after merging them. Let me know if you missed that point or if it wasn't clear enough. I'll update the doc accordingly if the document needs more clarity.

 I missed this point, will be more careful.

**Comment by yedhink** [comment]
>Can we modify this section of RBB to incorporate this suggestion?

Yes this can be done. Please raise an issue for it in BBA. We can take it up when someone is free.
